components/prombench/manifests/benchmark/3_prometheus-test.yaml

```
@@ -733,26 +733,39 @@ spec:
                 - prometheus
       securityContext:
         runAsUser: 0
+      initContainers:
+      - name: prometheus-builder
+        image: docker.io/geekodour/prometheus-builder:2.2.0
```

@sipian could you push a new image with the changes from this PR to the prombench docker repo.
If the changes are only with some refactoring the new tag could be `docker.io/prombench/prometheus-builder:2.0.1`

Than this can be changed to

```suggestion
        image: docker.io/prombench/prometheus-builder:2.0.1
```

At some point to make test changes easier maybe can even give access to the docker repo to @geekodour if he promises to not break anything :smile: 
The important thing would be to push new images under new tags and NOT overwrite anything that is currently in use in production.

---

lets leave it unresolved until @sipian confirms new image has been pushed and tested.